### PR TITLE
Move Client Code into an `ops` subdir

### LINKED ISSUE #965: Move frontend code into `clients/ops/`
### Is your feature request related to a specific problem?

Prepping for repo merger

### Describe the solution you'd like

Nesting the UI code down one level into an `ops` subdir

### PR DESCRIPTION
**Pull request by ThomasLaPiana**
# Purpose

Move client code into an `ops` subdir in prep for the repo merge

# Changes
* [x]  nest the client code into `ops`
* [x]  update all path references
* [x] update workflow tests

# Checklist
- [x] Update [`CHANGELOG.md`](https://github.com/ethyca/fidesops/blob/main/CHANGELOG.md) file
  - [x] Merge in main so the most recent `CHANGELOG.md` file is being appended to
  - [x] Add description within the `Unreleased` section in an appropriate category. Add a new category from the list at the top of the file if the needed one isn't already there.
  - [x] Add a link to this PR at the end of the description with the PR number as the text. example: [#1](https://github.com/ethyca/fidesops/pull/1)
- [x] Applicable documentation updated (guides, quickstart, postman collections, tutorial, fidesdemo, [database diagram](https://github.com/ethyca/fidesops/blob/main/docs/fidesops/docs/development/update_erd_diagram.md).
- If docs updated (select one):
  - [x] documentation complete, or draft/outline provided (tag docs-team to complete/review on this branch)
  - [ ] documentation issue created (tag docs-team to complete issue separately)
- [ ] Good unit test/integration test coverage
- [ ] This PR contains a DB migration. If checked, the reviewer should confirm with the author that the [down_revision correctly references the previous migration](https://ethyca.github.io/fidesops/development/contributing_details/#alembic-migrations) before merging
- [ ] The `Run Unsafe PR Checks` label has been applied, and checks have passed, if this PR touches any external services

# Ticket

Fixes #965

### CODE REVIEW

**Comment by ThomasLaPiana** [comment]
the meaningful files to review here are:
* dockerfile
* package.json
* .dockerignore
* docs files that got updated

File: .github/dependabot.yaml [hunk 1]
```diff
@@ -1,34 +1,34 @@
...
+  - package-ecosystem: npm
+    directory: "/clients/ops/admin-ui"
+    schedule:
+      interval: weekly
+    labels:
+      - dependencies
+      - javascript
+      - dependabot
+  - package-ecosystem: npm
+    directory: "/clients/privacy-center"
```
Comment (sanders41) [inline .github/dependabot.yaml:20]:
```suggestion
    directory: “/clients/opts/privacy-center”
```

Comment (ThomasLaPiana) [inline .github/dependabot.yaml:20]:
```suggestion
    directory: "/clients/ops/privacy-center"
```

**Review by sanders41 — APPROVED** [review]
lgtm. I know @chriscalhoun1974 was adding some configs for relative paths on the front end. Let's just wait to merge until he has a chance to look at it to make sure it won't break anything he is doing there.

**Comment by sanders41** [comment]
This integration test failure is different than the flaky one from before, but the test was passing before and the latest change you made couldn't possibly make it fail. I wonder if this has turned up a new flaky test. I'm going to restart it to see what happens.

**Review by chriscalhoun1974 — APPROVED** [review]
LGTM. My recent changes were to alias relative paths for import statements. Specifically the admin-ui react project.

**Pull request merged**